src/code/z_scene.c

```diff
@@ -3,116 +3,133 @@
 
 RomFile sNaviMsgFiles[];
 
-s32 Object_Spawn(ObjectContext* objectCtx, s16 objectId) {
+/**
+ * Spawn an object file of a specified ID that will persist through room changes.
+ *
+ * This only starts loading the object, the data will not be available immediately when the function returns.
```

Comment:
This function should fully load the object afaict, since it uses `DmaMgr_SendRequest1` which blocks the thread until the dma is done. And I don't think there is anything else that would prevent the object from being available immediately (?)

Reply:
My mistake :fearful: